week-2/errors/0.js

```diff
@@ -3,7 +3,13 @@
 // then call the function capitalise with a string input
 // interpret the error message and figure out why it's happening, if your prediction was wrong
 
+let str= "rabia"
 function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+    str = `${str[0].toUpperCase()}${str.slice(1)}`;
   return str;
 }
+
+console.log(capitalise (str));
+
+// *First it didn't work because Identifier 'str' has already been declared in function. 
```

Comment:
The first part of this explination is really good! You didn't need to add the declaration of the str var above though.  The problem is just that str has been implictly declared when it's passed as an arguement to the function and you then can't use let to declare str again inside the function, so removing let was all you needed to do